modules/utils/validation.js

```diff
@@ -163,18 +170,19 @@ function validateRule (item, path, itemId, meta, c) {
     properties = properties.set("operatorOptions", operatorOptions);
   }
 
+  // TODO PZ: fix validation
```

Comment:
Do the TODO?

Reply:
This is more complicated, this piece of code supposed to fix the query, but caused some bugs when using new query model. It's not obvious to me how it works and I don't know how to fix it now. I would have to spend another hours to invesitgate it, I think we can leave it as it is and fix it later, if needed.